packages/formation-react/src/components/AlertBox/AlertBox.jsx

```diff
@@ -126,12 +125,18 @@ AlertBox.propTypes = {
    * accented left edge or an icon
    */
   backgroundOnly: PropTypes.bool,
+
+  /**
+   * The header level to use with the headline prop
+   */
+  level: PropTypes.number,
```

Comment:
my only suggestion would be to make the PropType for `level` a function that checks to see that the value is a number from 1-6.